src/components/forms/VpnServerForm.js

```diff
@@ -0,0 +1,417 @@
+    } else if (name === "certificate_crn") {
+      newState.certificate_crn = crnList;
+    } else if (name === "client_ca_crn") {
+      newState.client_ca_crn = crnList;
+    } else {
+      newState = { [name]: value };
+    }
+    console.log(newState);
+    this.setState(newState);
+  }
+
+  /**
+   * handle input change of number-only fields
+   * @param {event} event
+   */
+  handleNumberInputChange(event) {
+    let value = parseInt(event.target.value) || null;
+    if (value || event.target.value === "") {
+      this.setState({ [event.target.name]: value });
+    }
+  }
+
+  /**
+   * Handle input change for  client_dns_server_ips text field
+   * @param {event} event
+   */
+  handleAllowedIps(event) {
+    // removing white space and checking for empty value
+    let value = event.target.value.replace(/\s*/g, "");
+    if (value === "") value = null;
+    this.setState({ client_dns_server_ips: value });
+  }
+
+  handleMultiSelectChange(name, value) {
+    this.setState(this.setNameToValue(name, value));
+  }
+
+  handleToggle(name) {
+    this.setState(this.toggleStateBoolean(name, this.state));
+  }
+
+  render() {
+    let composedId = `vpn-server-form-${this.props.data.name}`;
+    let classNameModalCheck = this.props.isModal
+      ? "fieldWidthSmaller"
+      : "fieldWidth";
+    let routeProps = {
+      invalidCallback: this.props.invalidVpnServerRouteCallback,
+      invalidTextCallback: this.props.invalidVpnServerRouteTextCallback,
+      arrayParentName: this.props.data.name,
+      parent_name: this.props.data.name,
+    };
+    transpose({ ...this.props.vpnServerRouteProps }, routeProps);
+    return (
+      <>
+        <IcseFormGroup>
+          {/* vpn server name */}
+          <IcseNameInput
+            id={composedId}
+            componentName={composedId}
+            value={this.state.name}
+            onChange={this.handleInputChange}
+            invalidCallback={() =>
+              this.props.invalidCallback(this.state, this.props)
+            }
+            invalidText={this.props.invalidTextCallback(this.state, this.props)}
+            hideHelperText
+            className={classNameModalCheck}
+          />
+          {/* resource group */}
+          <IcseSelect
+            formName={this.props.data.name + "-vpn-server-resource-group"}
+            name="resource_group"
+            labelText="Resource Group"
+            groups={this.props.resourceGroups}
+            value={this.state.resource_group}
+            handleInputChange={this.handleInputChange}
+            className={classNameModalCheck}
+          />
+          {/* vpc */}
+          <IcseSelect
+            formName={this.props.data.name + "-vpn-server-vpc"}
+            name="vpc"
+            labelText="VPC"
+            groups={this.props.vpcList}
+            value={this.state.vpc}
+            handleInputChange={this.handleInputChange}
+            invalid={checkNullorEmptyString(this.state.vpc)}
```

Comment:
let's use `isNullOrEmptyString` from lazy-z for all of these, we're using two different functions for the same thing (dropdowns below uses isNullOrEmptyString) so we should standardize